### ISSUE TITLE
NTR: Maxilla biopsy: multinuclear giant cells (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9184]
New term request Maxilla biopsy: multinuclear giant cells (elthat_100829154701): Maxillary; jaw

**Comment by pnrobinson** [comment]
@MickeySegal

are we talking about this?
https://jmedicalcasereports.biomedcentral.com/articles/10.1186/s13256-016-1173-3

**Comment by MickeySegal** [comment]
That looks like the same phenomenon.

Also https://www.ncbi.nlm.nih.gov/books/NBK1137/ "Histologic manifestations of lesions in the mandible and/or maxilla: non-neoplastic fibrotic lesions that contain numerous multinuclear giant cells and occasionally cysts. "

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000715
- label: Central giant cell lesion of the jaw
- parents: [Abnormal jaw morphology (HP:0030791)](https://hpo.jax.org/app/browse/term/HP:0030791)
- pmid: PMID:20301316|PMID:28316283|PMID:29988246
- def: A benign proliferative and asymptomatic intraosseous lesion of the jaw formed by fibrous tissue that contains multiple focus of hemorrhage and an aggregation of multinucleated giant cells.
- comment: These lesions occur more frequently in the mandible than in the maxilla, and occur more frequently on the right side than on the left side. In the maxilla, the lesion may invade the maxillary sinus floor, the orbit and/or nasal passages, which can lead to facial asymmetry, nasal deviation, and mobility of associated dental elements. CGCLs can be classified as aggressive or non-aggressive.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9184
- synonyms:
